packages/analytics-js-plugins/__tests__/oneTrustConsentManager/index.test.ts

```diff
@@ -115,16 +118,16 @@ describe('Plugin - OneTrust', () => {
       key: 'value',
     };
 
-    const isDestinationConsented = OneTrust().consentProvider.isDestinationConsented(
+    const isDestinationConsented = OneTrustConsentManager().consentManager.isDestinationConsented(
       state,
       destConfig,
       defaultLogger,
     );
     expect(isDestinationConsented).toBeTruthy();
   });
   it('should return false if destination categories are not consented', () => {
-    state.consents.consentProviderInitialized.value = true;
-    state.consents.allowedConsents.value = {
+    state.consents.data.value.initialized = true;
```

Comment:
do not mutate the data object properties. you need to createnew data object and update it so refference is new and  any effect run. elsewise effects tracking the data object will not run